### PR TITLE
Added GetMORnPrefix to extract Rn string from Dn

### PR DESCRIPTION
**Pull request by sajagana**

### CODE REVIEW

**Review by akinross — APPROVED** [review]
LGTM

File: models/util.go [hunk 1]
```diff
@@ -94,3 +94,9 @@ func CurlyBraces(value string) string {
 		return value
 	}
 }
+
+func GetMORnPrefix(dn string) string {
```
Comment (shrsr) [inline models/util.go:98]: Does this function work for Rns containing '/'?

**Review by shrsr — CHANGES_REQUESTED** [review]

File: models/util.go (hunk 1 above)
Comment (shrsr) [reply models/util.go:98]: Screenshot 2023-09-12 at 9 40 44 AM

**Review by shrsr — APPROVED** [review]
LGTM

**Pull request merged**